### PR TITLE
Add unit test coverage to CI script

### LINKED ISSUE #64: Add coverage to CI
CI should fail if coverage is less than 80%

### PR DESCRIPTION
**Pull request by palemieux**
Closes #64

### CODE REVIEW

File: src/main/python/ttconv/imsc/style_properties.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def local_name(cls):
     raise NotImplementedError
 
   @classmethod
```
Comment (BobBrownConsulting) [inline src/main/python/ttconv/imsc/style_properties.py:57]: Fixing pylint unused variable

File: src/test/python/test_imsc_reader.py [hunk 2]
```diff
@@ -35,9 +35,52 @@
 import ttconv.model as model
 import ttconv.style_properties as styles
 import ttconv.imsc.reader as imsc_reader
+import ttconv.imsc.namespaces as xml_ns
 
 class IMSCReaderTest(unittest.TestCase):
```
Comment (BobBrownConsulting) [inline src/test/python/test_imsc_reader.py:41]: Not certain if this is needed. Happy to remove if not.

File: src/test/python/test_imsc_reader.py [hunk 3]
```diff
@@ -35,9 +35,52 @@
...
 
+  def setUp(self):
+    et.register_namespace("ttml", xml_ns.TTML)
+    et.register_namespace("ttp", xml_ns.TTP)
+    et.register_namespace("tts", xml_ns.TTS)
+    et.register_namespace("ittp", xml_ns.ITTP)
+    et.register_namespace("itts", xml_ns.ITTS)
+
+  def test_reader_tt_element_not_root_element(self):
+
```
Comment (BobBrownConsulting) [inline src/test/python/test_imsc_reader.py:50]: Covers first early return from reader.

File: src/test/python/test_imsc_reader.py [hunk 4]
```diff
@@ -35,9 +35,52 @@
...
+          xmlns:ttm="http://www.w3.org/ns/ttml#metadata" 
+          xmlns:tts="http://www.w3.org/ns/ttml#styling"
+          xmlns:ttp="http://www.w3.org/ns/ttml#parameter" 
+          xmlns:ittp="http://www.w3.org/ns/ttml/profile/imsc1#parameter"
+          ittp:activeArea="50% 50% 80% 80%"
+          tts:extent="640px 480px"
+          ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1/text">
+          <asdf></asdf>
+      </tt>"""
+
```
Comment (BobBrownConsulting) [inline src/test/python/test_imsc_reader.py:80]: This XML fails as <asdf> is not a valid TT value. The reader throws. 
Is the second None return in reader even valid?
What XML would trigger that case?
It seems the reader throws in many cases of invalid XML.

Comment (palemieux) [reply src/test/python/test_imsc_reader.py:80]: Replace `<tt>` with `<foo>`, i.e. `None` will be returned if the root element is not `<tt>`

Comment (palemieux) [reply src/test/python/test_imsc_reader.py:80]: Oh... this is already covered by `test_reader_tt_element_not_root_element`. I suggest removing this test.

File: src/test/python/test_imsc_reader.py (hunk 2 above)
Comment (palemieux) [reply src/test/python/test_imsc_reader.py:41]: What are you referring to?

Comment (BobBrownConsulting) [reply src/test/python/test_imsc_reader.py:41]: def setUp(self):

File: src/test/python/test_imsc_reader.py (hunk 4 above)
Comment (BobBrownConsulting) [reply src/test/python/test_imsc_reader.py]: yes, that is correct. I wasn't sure if there was another way to pass the first check in to_model while still triggering the second check. I tried several things but the to_model alway threw exceptions which is the right way to do it.

**Review by BobBrownConsulting — APPROVED** [review]

File: src/test/python/test_imsc_reader.py (hunk 2 above)
Comment (palemieux) [reply src/test/python/test_imsc_reader.py:41]: Already defined in `imsc_writer`

**Pull request merged**